rts/at/tx/src/main/java/org/jboss/jbossts/star/service/TMApplication.java

```diff
@@ -46,7 +46,17 @@ public class TMApplication extends Application {
     HashSet<Object> singletons = new HashSet<Object>();
     Set<Class<?>> classes = new HashSet<Class<?>> ();
 
+
```

Comment:
You added an extra blank line, the policy is that we should match the existing format in the file.